sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuSemaphore.scala

```diff
@@ -325,6 +337,7 @@ private final class GpuSemaphore() extends Logging {
   type GpuBackingSemaphore = PrioritySemaphore[Long]
   private val semaphore = new GpuBackingSemaphore(MAX_PERMITS)
   // Keep track of all tasks that are both active on the GPU and blocked waiting on the GPU
+  // taskAttemptId => semaphoreTaskInfo
```

Comment:
It took me a moment to realize that this comment illustrates the mapping from taskAttempt to semaphoreTaskInfo.  

On first read, I thought this was commented code. :p

Reply:
```
  // A map of taskAttemptId => semaphoreTaskInfo.
  // This map keeps track of all tasks that are both active on the GPU and blocked waiting
  // on the GPU.
```

Made it more obvious that it is a comment 🙂